sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -451,7 +457,7 @@ export class EventHubReceiver extends LinkEntity {
     } catch (err) {
       this.isConnecting = false;
       const error = translate(err);
-      logger.warning(
+      logger.error(
```

Comment:
I feel like it is an error if we can't connect! not a mere warning. What do you think?